css/main.css

```diff
@@ -104,9 +104,8 @@ input[type="text"] {
   width: 100%;
   padding: 1rem 0.75rem;
   background: white;
-  border: 0;
+  border: 1px solid #AAAAAA;
   border-radius: var(--border-radius);
-  outline: 2px solid transparent;
   transition: all 0.1s ease-out;
```

Comment:
@fbane removing the `outline` breaks the CSS transition on the `border` property on `:focus`

Reply:
Hmm ok. I don't recall what I was trying here, but reverted it.